docs/DEVELOPMENT_SOFTWARE_ESP.md

```diff
@@ -0,0 +1,18 @@
+# mLRS Documentation: Compiling and Flashing for ESP82xx and ESP32 Devices #
+
+([back to main page](../README.md))
+
+For ESP82xx and ESP32 devices, the project is using [PlatformIO](https://platformio.org/) in combination with [VSCode](https://code.visualstudio.com/).
+
+## Software: Installation Bits and Bops
+
+Let's assume that the project should be located in the folder C:/Me/Documents/Github/mlrs.
```

Comment:
Should this include how to install VSCode + PlatformIO?  Similar to main repo?

Reply:
at some point, yes, sure, I think